setup.cfg

```
@@ -32,15 +32,19 @@ verbose = 2
 format = pylint
 # see: https://www.flake8rules.com/
 ignore =
-    E731  # Do not assign a lambda expression, use a def
-    E231  # Ignore missing space after comma
-    W504  # Line break occurred after a binary operator
-    W605  # Invalid escape sequence 'x'
+    E129  # Ignore "Visually indented line with same indent as next logical line"
+    E231  # Ignore "Missing whitespace after ',', ';', or ':'"
+    E731  # Ignore "Do not assign a lambda expression, use a def"
+    W503  # Ignore "Line break occurred before a binary operator"
+    W504  # Ignore "Line break occurred after a binary operator"
+    W605  # Ignore "Invalid escape sequence 'x'"
 
 [yapf]
 based_on_style = pep8
 spaces_before_comment = 2
 split_before_logical_operator = true
+split_before_arithmetic_operator = true
+split_before_bitwise_operator = true
 COLUMN_LIMIT = 120
```

If we ignore both `W503` and `W504`, we can just remove these additions.
```suggestion
split_before_logical_operator = true
COLUMN_LIMIT = 120
```